backend/cmm_helpers.ml

```diff
@@ -1214,6 +1220,17 @@ let rec low_32 dbg = function
   | Clet (id, e, body) -> Clet (id, e, low_32 dbg body)
   | x -> x
 
+(* Like [low_32] but for 63-bit integers held in 64-bit registers. *)
+let rec low_63 dbg e =
+  check_64_bit_target "low_63";
+  match e with
+  | Cop (Casr, [Cop (Clsl, [x; Cconst_int (1, _)], _); Cconst_int (1, _)], _) ->
+    low_63 dbg x
+  | Cop (Cand, [x; Cconst_natint (0x7FFF_FFFF_FFFF_FFFFn, _)], _) ->
+    low_63 dbg x
+  | Clet (id, x, body) -> Clet (id, x, low_63 dbg body)
```

Comment:
Why not use `Cmm.map_tail` here ? It seems designed for that kind of thing (and covers more cases than just `Clet`).

Reply:
It matches `low_32`, which is the important thing for the moment.  I've added a CR that this could maybe use `map_tail`.